edgexfoundry/app-service-configurable
# build: Update go.mod to use latest go-mod-bootstrap properly

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A for Build PR**
- [ ] I have opened a PR for the related docs change (if not, why?)  **N/A for Build PR**


## Testing Instructions
See instructions here: https://github.com/edgexfoundry/app-functions-sdk-go/pull/1038

## New Dependency Instructions (If applicable)
**N/A**
